src/main/kotlin/app/revanced/patches/youtube/layout/branding/header/PremiumHeadingPatch.kt

```diff
@@ -32,9 +32,9 @@ object PremiumHeadingPatch : ResourcePatch() {
         val resDirectory = context["res"]
 
         val (original, replacement) = if (usePremiumHeading!!)
-            DEFAULT_HEADING_RES to PREMIUM_HEADING_RES
-        else
             PREMIUM_HEADING_RES to DEFAULT_HEADING_RES
+        else
+            DEFAULT_HEADING_RES to PREMIUM_HEADING_RES
```

Comment:
I do not quite understand, if usePremiumHeading is false, then it should be premium (original) -> default (replacement), but this PR inverts this logic?

Reply:
`original` is the source resource, and `replacement` is the one getting replaced. Maybe those can be renamed to `source` and `target` to avoid confusions?